packages/manager/modules/emailpro/src/account/emailpro-account.controller.js

```diff
@@ -254,6 +256,7 @@ export default /* @ngInject */ (
       primaryEmailAddress: newConfiguredAccount,
     })
       .then((account) => {
+        $scope.accountIds = drop($scope.accountIds, 1);
```

Comment:
Maybe we can avoid to import `lodash/drop` ?

```suggestion
        $scope.accountIds = $scope.accountIds.slice(1);
```